config/locales/en/helpers.yml

```diff
@@ -133,6 +133,31 @@ en:
         conviction: How old were you when you got convicted?
       steps_conviction_conviction_type_form:
         conviction_type: What type of conviction did you get?
+      steps_conviction_known_date_form:
```

Comment:
I think this whole block needs to be removed too, probably again a bad merge messing around.